pkg/util/common.go

```diff
@@ -0,0 +1,96 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"bufio"
+	"errors"
+	"log"
+	"os"
+	"os/exec"
+	"strings"
+)
+
+// Run wraps the exec.Cmd.Run() command and sets the standard output.
+// TODO: Should this take an error code argument/return an error code?
```

Comment:
This a great thing to point out and I definitely think `Run()` should be returning error here. Otherwise, any user of this package is unable to determine whether to exit or continue as they see fit based on the outcome of calling this function. While it doesn't make a huge difference for `ff`, it could prove difficult to work with if any other tool consumes it.